redirects.ts

```diff
@@ -138,22 +138,25 @@ export const redirectsOptions: Options = {
             from: '/general/community-resources/token-delivery-and-kyc', // #200 Restructure docs
         },
         {
-            to: '/dapp/',
+            to: '/build/',
             from: [
                 '/oasis-sdk/', // #200 Restructure docs
```

Comment:
You will need to update a series of redirects below. e.g.

```suggestion
                '/dapp/', // #1058 Rename dapp to build
                '/oasis-sdk/', // #200 Restructure docs
```